src/aat/resources/features/F-106 - Update Supplementary Data/S-613.td.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "CAA must NOT be able to Update supplementary data of any PRIVATE case that requires explicit access",
+  "_guid_": "S-613",
+  "productName": "CCD Data Store api",
+  "operationName": "Update Supplementary Data",
```

Comment:
This can be removed, as the base file contains this information.

Reply:
This test has failed after removing these lines, as seen [here](https://static-build.platform.hmcts.net/static-files/STHxh1XUG45_edsB3d1IzI9mNCxdn4xR0W9D8lie7DYxNjc0NDk1NDk5MDAxOjI2Okx1Y2lhbm8uTWFyc2lsaW9ASE1DVFMuTkVUOmpvYi9ITUNUU19hX3RvX2Mvam9iL2NjZC1kYXRhLXN0b3JlLWFwaS92aWV3L2NoYW5nZS1yZXF1ZXN0cy9qb2IvUFItMjE3OC81Ni9hcnRpZmFjdA==/BEFTA%20Report%20for%20Functional%20Tests/report-feature_53_3676448755.html). S-612 is a similar case, where those lines are present in the main and base test files. Should we keep them in this case?